pkg/util/filter/filter_function.go

```diff
@@ -0,0 +1,3 @@
+package filter
+
+type Func func(string) bool
```

Comment:
Is it possible to either move this somewhere relevant like chunks or deletion package? It looks out of place to me.
The other option is to add more context to it if moving it is not possible due to circular dependency.